### ISSUE TITLE
[BUG] (Accessibility) Lightmode export dropdown has a conflicting background

### ISSUE THREAD

**Issue by mnyamunda-scottlogic** [ScottLogic/sl-tech-carbon-estimator#232]
**Describe the bug**
When in light mode, text correctly turns black, but background is also a dark blue, which causes contrast accessibility issues.
**To Reproduce**
1. Go to local branch "develop" with `npm run start:npm-package`
2. Enable lightmode through either system or browser settings
3. Run a calculation 
4. Click export dropdown

**Expected behaviour**
Dark text light background 

**Actual Behaviour**
Dark text dark background:

<img width="217" height="185" alt="Image" src="https://github.com/user-attachments/assets/581a140d-f888-4ce5-aa88-6a2a641881fe" />
